src/models/split_transfers.py

```diff
@@ -141,19 +150,23 @@ def process(
         so that any overdraft from slippage can be carried over and deducted from
         the COW rewards.
         """
-        penalty_total = self._process_native_transfers(
+        total_penalty = self._process_native_transfers(
             indexed_slippage=index_by(
                 slippages.solvers_with_negative_total, "solver_address"
             )
         )
+        self.log_saver.print(
+            f"Total Negative Slippage (ETH): {total_penalty / 10**18}",
```

Comment:
I'm not sure how much we need the decimal accuracy here, but rounding this up makes this easier to read.

Reply:
I could make is ~ 4 decimal places.

Reply:
Unfortunately I think doing this might wind up being a bit of a messy change (adding some `format_decimal()` function call in several places throughout the code... I will make an attempt here, but if it gets out of hand I might revert.

Reply:
Turned out alright.

Reply:
> I could make is ~ 4 decimal places.

Thanks. Look great!